### PR TITLE
docs: add mention about dagger login on cloud-get-started documentation

### LINKED ISSUE #7734: `dagger login` is not mentioned as a way to send traces to cloud
### What is the issue?

We've recently introduced `dagger login` as a simplified way for users to send local traces to cloud. Docs need an update here https://docs.dagger.io/manuals/user/cloud-get-started#connect-from-your-local-development-host to reflect that.

### PR DESCRIPTION
**Pull request by mcruzdev**
## What?

This pull request aims to mention `dagger login` at [Dagger Cloud Introduction](https://docs.dagger.io/manuals/user/cloud-get-started/#connect-from-your-local-development-host) documentation.

## Why?

After the `dagger login` there was no mentions about `dagger login` on Dagger Cloud Introduction documentation.

The idea is to add it (`dagger login`) as a new way to connect to Dagger Cloud.

Closes #7734

### CODE REVIEW

**Comment by marcosnils** [comment]
cc @vikram-dagger

File: docs/current_docs/manuals/user/visualization/cloud-get-started.mdx [hunk 1]
```diff
@@ -76,12 +76,26 @@ Once you have your token, you can now use it to connect Dagger Cloud with your C
 
 You can visualize and debug your local Dagger pipeline runs with Dagger Cloud to identify issues before pushing them to CI.
 
-To configure for local development, set your Dagger Cloud token as an environment variable. For example:
+To configure for local development, run `dagger login` command.
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/visualization/cloud-get-started.mdx:79]:
```suggestion
To configure for local development, run the `dagger login` command.
```

File: docs/current_docs/manuals/user/visualization/cloud-get-started.mdx [hunk 2]
```diff
@@ -76,12 +76,26 @@ Once you have your token, you can now use it to connect Dagger Cloud with your C
...
+
+The Dagger CLI will invite you to authenticate your device by displaying a link containing a unique key. Click the link in your browser, and verify that you see the same key in the Dagger Cloud Web interface.
 
 ```shell
-export DAGGER_CLOUD_TOKEN={your token}
+$ dagger login
+Browser opened to: https://auth.dagger.cloud/activate?user_code=FQNJ-BANG
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/visualization/cloud-get-started.mdx:85]:
```suggestion
Browser opened to: https://auth.dagger.cloud/activate?user_code=XXXX-YYYY
```

File: docs/current_docs/manuals/user/visualization/cloud-get-started.mdx [hunk 3]
```diff
@@ -76,12 +76,26 @@ Once you have your token, you can now use it to connect Dagger Cloud with your C
...
+Confirmation code: FQNJ-BANG
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/visualization/cloud-get-started.mdx:86]:
```suggestion
Confirmation code: XXXX-YYYY
```

**Review by vikram-dagger — CHANGES_REQUESTED** [review]
Thanks for this. See minor inline suggestions.

**Review by vikram-dagger — APPROVED** [review]

**Pull request merged**